diff --git a/stdlib.go b/stdlib.go
index 0cbf361..98da78e 100644
--- a/stdlib.go
+++ b/stdlib.go
@@ -828,6 +828,9 @@ const StdLib = "#!/usr/bin/env bash\n" +
 	"  elif [[ -f $HOME/.direnvrc ]]; then\n" +
 	"    # shellcheck disable=SC1090\n" +
 	"    source \"$HOME/.direnvrc\" >&2\n" +
+	"  elif [[ -f @direnvrc@ ]]; then\n" +
+	"    # shellcheck disable=SC1090\n" +
+	"    source \"@direnvrc@\" >&2\n" +
 	"  fi\n" +
 	"\n" +
 	"  # and finally load the .envrc\n" +
