diff --git a/REQUIRE b/REQUIRE
index 507d075..61b8e45 100644
--- a/REQUIRE
+++ b/REQUIRE
@@ -3,5 +3,4 @@ Compat 0.52.0
 Colors
 BinDeps 0.3.21
 Graphics 0.1
-@osx Homebrew
 @windows WinRPM
diff --git a/deps/build.jl b/deps/build.jl
index 7d93002..76a7882 100644
--- a/deps/build.jl
+++ b/deps/build.jl
@@ -40,24 +40,6 @@ if Sys.iswindows()
 end
 
 if is_apple()
-    if Pkg.installed("Homebrew") === nothing
-        error("Homebrew package not installed, please run Pkg.add(\"Homebrew\")")
-    end
-    using Homebrew
-    provides( Homebrew.HB, "cairo", cairo, os = :Darwin )
-    provides( Homebrew.HB, "pango", [pango, pangocairo], os = :Darwin, onload =
-    """
-    function __init__()
-        ENV["PANGO_SYSCONFDIR"] = joinpath("$(Homebrew.prefix())", "etc")
-    end
-    """ )
-    provides( Homebrew.HB, "fontconfig", fontconfig, os = :Darwin )
-    provides( Homebrew.HB, "glib", gobject, os = :Darwin )
-    provides( Homebrew.HB, "libpng", libpng, os = :Darwin )
-    provides( Homebrew.HB, "gettext", gettext, os = :Darwin )
-    provides( Homebrew.HB, "freetype", freetype, os = :Darwin )
-    provides( Homebrew.HB, "libffi", libffi, os = :Darwin )
-    provides( Homebrew.HB, "pixman", pixman, os = :Darwin )
 end
 
 # System Package Managers
