diff --git a/REQUIRE b/REQUIRE
index 60d25d1..6ea9e39 100644
--- a/REQUIRE
+++ b/REQUIRE
@@ -1,5 +1,4 @@
 julia 0.6
 BinDeps 0.3.21
-@osx Homebrew
 Compat 0.41.0
 @windows WinRPM
diff --git a/deps/build.jl b/deps/build.jl
index 4fbd7ae..987f585 100644
--- a/deps/build.jl
+++ b/deps/build.jl
@@ -30,8 +30,6 @@ if Compat.Sys.iswindows()
     using WinRPM
     provides(WinRPM.RPM, "zeromq", [zmq], os = :Windows)
 elseif Compat.Sys.isapple()
-    using Homebrew
-    provides(Homebrew.HB, "zeromq@3.2", zmq, os = :Darwin)
 end
 
 @BinDeps.install Dict(:zmq => :zmq)
