diff --git a/colors/srcery.vim b/colors/srcery.vim
index d35fa61..0dd56d4 100644
--- a/colors/srcery.vim
+++ b/colors/srcery.vim
@@ -22,22 +22,22 @@ endif
 
 " Palette {{{
 
-let s:black          = ['#1C1B19', 0]
-let s:red            = ['#EF2F27', 1]
-let s:green          = ['#519F50', 2]
-let s:yellow         = ['#FBB829', 3]
-let s:blue           = ['#2C78BF', 4]
-let s:magenta        = ['#E02C6D', 5]
-let s:cyan           = ['#0AAEB3', 6]
-let s:white          = ['#D0BFA1', 7]
-let s:bright_black   = ['#918175', 8]
-let s:bright_red     = ['#F75341', 9]
-let s:bright_green   = ['#98BC37', 10]
-let s:bright_yellow  = ['#FED06E', 11]
-let s:bright_blue    = ['#68A8E4', 12]
-let s:bright_magenta = ['#FF5C8F', 13]
-let s:bright_cyan    = ['#53FDE9', 14]
-let s:bright_white   = ['#FCE8C3', 15]
+let s:black          = ['@black_hex@', @black_nr@]
+let s:red            = ['@red_hex@', @red_nr@]
+let s:green          = ['@green_hex@', @green_nr@]
+let s:yellow         = ['@yellow_hex@', @yellow_nr@]
+let s:blue           = ['@blue_hex@', @blue_nr@]
+let s:magenta        = ['@magenta_hex@', @magenta_nr@]
+let s:cyan           = ['@cyan_hex@', @cyan_nr@]
+let s:white          = ['@white_hex@', @white_nr@]
+let s:bright_black   = ['@brblack_hex@', @brblack_nr@]
+let s:bright_red     = ['@brred_hex@', @brred_nr@]
+let s:bright_green   = ['@brgreen_hex@', @brgreen_nr@]
+let s:bright_yellow  = ['@bryellow_hex@', @bryellow_nr@]
+let s:bright_blue    = ['@brblue_hex@', @brblue_nr@]
+let s:bright_magenta = ['@brmagenta_hex@', @brmagenta_nr@]
+let s:bright_cyan    = ['@brcyan_hex@', @brcyan_nr@]
+let s:bright_white   = ['@brwhite_hex@', @brwhite_nr@]
 
 " xterm colors.
 let s:orange        = ['#FF5F00', 202]
